### DISCUSSION TITLE
Providing one-off support without configuring network settings

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk-server#393] [Q&A]
Unless I'm missing something obvious there appears to be no way of providing one-off support without having the end user configure the network ID / relay server settings.

Is there a way around this?

**Comment by ghost** [comment]
Anyone?

**Comment by GuillaumeHullin** [comment]
https://rustdesk.com/docs/en/self-host/client-configuration/#5-put-config-in-rustdeskexe-file-name-windows-only

Here you go

**Reply by ghost** [reply]
That only works for Windows machines. 

I think the devs really need to rethink the client, deployment and what I would consider as normal use-cases such as my original question. 

The sooner msi and pkg files are the norm the better for the project.

**Reply by GuillaumeHullin** [reply]
Well basically, the way to go for easier deploy is the Custom client generator... but that's the Pro version of RustDesk.

Otherwise you can do with scripting: https://rustdesk.com/docs/en/self-host/client-deployment/

**Reply by bigretromike** [reply]
he did, pro version does what you are asking.
